src/locales/de.ts

```diff
@@ -10,42 +10,42 @@ export default {
     'general.signOut': 'Abmelden',
     'general.save': 'Speichern',
     'general.default': 'Standard',
-    'general.networkError': 'Netzwerk-Fehler',
+    'general.networkError': 'Netzwerkfehler',
 
     // Storage
-    'storage.name': 'Speicherort Name',
+    'storage.name': 'Name des Speicherorts',
     'storage.noStorage': 'Kein Speicherort',
     'storage.create': 'Speicherort erstellen',
     'storage.edit': 'Speicherort ändern',
     'storage.rename': 'Speicherort umbenennen',
-    'storage.renameMessage': 'Name für Speicherort ändern',
+    'storage.renameMessage': 'Name des Speicherorts ändern',
     'storage.remove': 'Speicherort entfernen',
-    'storage.removeMessage': 'Speicherort wird von App entknüpft',
+    'storage.removeMessage': 'Speicherort wird gelöscht',
     'storage.delete': 'Speicherort löschen',
-    'storage.move': 'Notiz verschieben',
-    'storage.moveTitle': 'Notiz in anderen Speicherort verschieben',
+    'storage.move': 'Speicherort verschieben',
+    'storage.moveTitle': 'Speicherort wird verschoben',
     'storage.moveMessage':
-      'Es wird versucht eine Notiz in einen anderen Speicherort zu verschieben.',
-    'storage.copy': 'Notiz kopieren',
+      'Es wird versucht, den Speicherort zu verschieben.',
+    'storage.copy': 'Speicherort kopieren',
     'storage.typeLocal': 'Lokal',
     'storage.typeCloud': 'Cloud',
     'storage.needSignIn':
-      'Du musst dich anmelden um einen Cloud-Speicher zu erstellen.',
+      'Du musst dich anmelden, um einen Cloud-Speicher zu erstellen.',
     'storage.syncDate': 'Zuletzt synchronisiert am',
 
     //Folder
     'folder.create': 'Neuer Ordner',
     'folder.rename': 'Ordner umbenennen',
     'folder.renameMessage':
-      'Gib den neuen Ordnernamen an, jede Notiz und Unterordner-Pfad wird aktualisiert.',
+      'Gib den neuen Ordnernamen an. Notizen und Pfade von Unterordnern werden automatisch aktualisiert.',
     'folder.renameErrorMessage': 'Ordner konnte nicht umbenannt werden',
     'folder.remove': 'Ordner löschen',
     'folder.removeMessage': 'Alle Notizen und Unterordner werden gelöscht.',
 
     //Tag
-    'tag.tag': 'Etikett',
-    'tag.remove': 'Etikett löschen',
-    'tag.removeMessage': 'Das Etikett wird von allen Notizen entfernt.',
+    'tag.tag': 'Tag',
+    'tag.remove': 'Tag löschen',
+    'tag.removeMessage': 'Der Tag wird von allen Notizen entfernt.',
```

Comment:
`Das Tag`

Reply:
Der and Das Tag is both correct according to the sources I looked at

Reply:
this is correct. This is an entirely personal preference. I prefer the latter.